### ISSUE TITLE
Accessibility: meet the WCAG standards for accessible text color

### ISSUE THREAD

**Issue by martineizayaga** [DivinumOfficium/divinum-officium#1665]
I ran an accessibility checkup on Firefox Dev Tools. I found out that the color contrast for the red color for text doesn't meet [the WCAG guidelines](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast) (the minimum ratio should be at least 4.5 : 1.

A color like #e00000 would work instead. The radio would be 5.04 : 1.
<img width="952" alt="Screen Shot 2020-05-05 at 1 18 16 PM" src="https://user-images.githubusercontent.com/28035767/81095487-10d72400-8ed3-11ea-88a6-7a6a9b731c7c.png">
<img width="952" alt="Screen Shot 2020-05-05 at 1 18 19 PM" src="https://user-images.githubusercontent.com/28035767/81095491-12085100-8ed3-11ea-8d72-2039ffc6a195.png">

**Comment by APMarcello3** [comment]
The color red is coded in simply as "red". I see no need to revise at this time. This is low priority.

**Comment by martineizayaga** [comment]
Increasing the ratio would help people that have visual impairments. I would be willing to work on it.

Another option is to change the background color.
